<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shares</class>
 <widget class="QDialog" name="shares">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>658</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Candlestick Chart</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(244, 244, 248);</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="_2">
     <item>
      <widget class="QLabel" name="stocknameLabel">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Apple</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="_3">
       <item>
        <widget class="QLabel" name="timeLabel">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>9:41:00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dateLabel">
         <property name="text">
          <string>Monday, June 8</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QDateTimeEdit" name="datefromInput">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2005</year>
         <month>7</month>
         <day>26</day>
        </datetime>
       </property>
       <property name="date">
        <date>
         <year>2005</year>
         <month>7</month>
         <day>26</day>
        </date>
       </property>
       <property name="displayFormat">
        <string>dd.MM.yyyy hh:mm:ss</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="datetoInput">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2023</year>
         <month>7</month>
         <day>26</day>
        </datetime>
       </property>
       <property name="date">
        <date>
         <year>2023</year>
         <month>7</month>
         <day>26</day>
        </date>
       </property>
       <property name="displayFormat">
        <string>dd.MM.yyyy hh:mm:ss</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="intervalLabel">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Options:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="smaCheckBox">
         <property name="text">
          <string>SMA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="emaCheckBox">
         <property name="text">
          <string>EMA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="intervalComboBox">
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updateButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&gt;&gt;&gt;</string>
         </property>
         <property name="shortcut">
          <string>Return</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QChartView" name="graphicsCandleView">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QChartView" name="graphicsLineView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
